engine/src/flutter/impeller/display_list/canvas.cc

```diff
@@ -1052,10 +1054,10 @@ void Canvas::SaveLayer(const Paint& paint,
   if (backdrop_filter) {
     local_position = subpass_coverage.GetOrigin() - GetGlobalPassPosition();
     Canvas::BackdropFilterProc backdrop_filter_proc =
-        [backdrop_filter = backdrop_filter](
+        [backdrop_filter = backdrop_filter, &canvas = *this](
```

Comment:
You are capturing a point to an object, there is no guarantee this won't be a dangling pointer.

Reply:
Added a note that I think this is safe because the bdf doesn't outlive the canvas instance